packages/query/src/queriers/tendermint.ts

```diff
@@ -10,6 +10,8 @@ const knownTendermintErrors = [
   'proof did not verify',
   'is not a valid field element',
   'is not a valid SCT root',
+  'cannot claim unbonding tokens before the end epoch',
+  'undelegation was prepared for next epoch',
```

Comment:
Unrelated to this PR, see https://github.com/penumbra-zone/penumbra/issues/3961. 

These messages are actually in the core repo — not Tendermint itself. Thus, they can (and probably should) be changed to include a machine-readable error code, rather than us having to parse human-readable text.